src/BurgerIcon.js

```diff
@@ -7,12 +7,11 @@ let BurgerIcon = React.createClass({
 
   getLineStyle(index) {
     return appendVendorPrefix({
-      position: 'fixed',
-      height: 6,
-      width: 36,
-      left: 36,
-      top: 36 + 12 * index,
-      zIndex: 1,
+      position: 'absolute',
+      height: "15%",
+      left: 0,
+      right: 0,
+      top: (33 * index) + "%",
```

Comment:
We could avoid a gap below the third bar by using 20% instead of 15%. So this would become:
```js
position: 'absolute',
height: '20%',
left: 0,
right: 0,
top: 20 * (index * 2) + '%',
```